packages/next/src/build/webpack-config.ts

```diff
@@ -1390,41 +1404,59 @@ export default async function getBaseWebpackConfig(
     // Also disable esm request when appDir is enabled
     const isEsmRequested = dependencyType === 'esm'
 
+    /**
+     * @param {string} localRes the full path to the file
```

Comment:
You can leave out the types in the tsdoc comment I think 👀

Reply:
copilot 👀